files/spark/conf/spark-defaults.conf

```
@@ -0,0 +1 @@
+spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04
```

Shouldn't this be 

```suggestion
spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04_{risk}
```

using either edge or stable?

---

done